### PR TITLE
ADJUST1-475 Allowing adjustment types to be changed

### PR DESCRIPTION
**Pull request by pavilionsahota**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/legacy/controller/LegacyController.kt [hunk 1]
```diff
@@ -110,7 +110,7 @@ class LegacyController(
     @RequestBody adjustment: LegacyAdjustment,
   ) {
     legacyService.update(adjustmentId, adjustment).also {
-      eventService.update(adjustmentId, adjustment.offenderNo, AdjustmentSource.NOMIS)
+      eventService.update(adjustmentId, adjustment.offenderNo, AdjustmentSource.NOMIS, legacyService.transform(adjustment.adjustmentType))
```
Comment (ldlharper) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/legacy/controller/LegacyController.kt:113]: Can you revert this change. We don't need to pass the adjustment type into the event service here.

**Comment by ldlharper** [comment]
Can you rebase the branch so that you have the changes from dev, we want to make sure the build is passing? (I'd just do a hard reset and cherry-pick your commit.)

File: src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/legacy/controller/LegacyController.kt (hunk 1 above)
Comment (pavilionsahota) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/legacy/controller/LegacyController.kt]: I've reverted this change

**Review by ldlharper — APPROVED** [review]
LGTM
